server/profileFormSubmission.js

```diff
@@ -8,25 +11,52 @@ const pool = new Pool({
     port: process.env.port,
 });
 
+s3.config.update({
+    accessKeyId: "AKIA47HST523YVS7DQNW",  
+    secretAccessKey: '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',
+    region: "us-west-1",
+});
+
+const LICENSE_BUCKET = "cleaner-licenses";
+const TOOLS_BUCKET = "cleaner-tools";
 
-module.exports.profileInsertion = async (event, callback )=> {
+let s3Urls = ['',''];
+
+const s3Params = (bucket_name, profileId) => {
+    return {
+        ACL: 'public-read',
+        Bucket: bucket_name,
+        ContentType: 'application/pdf',
+        Key: `${profileId}.pdf`,
```

Comment:
We should try to assign more meaningful names to these files; maybe append profileId to the name of the document uploaded (ex: 1-resume2020.pdf)

Even better would be allowing users to select from a dropdown or smth to categorize what type of file they're uploading (will bring up during retro/future sprint plan)

Reply:
that's a great idea!